pandas/tools/pivot.py

```diff
@@ -381,8 +381,11 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
         Values to group by in the columns
     values : array-like, optional
         Array of values to aggregate according to the factors
-    aggfunc : function, optional
+    aggfunc : function, or string, optional
         If no values array is passed, computes a frequency table
+        If 'pct' is passed, will normalize frequency table.
```

Comment:
It may better to describe normalization is performed for the entire level (not for sub-levels) in ``MultiIndex`` case.